packages/wallet-sdk/src/triplesec.ts

```diff
@@ -0,0 +1,17 @@
+/* eslint-disable node/prefer-global/buffer */
+import * as triplesec from 'triplesec';
+
+export function triplesecDecrypt(
+  arg: { data: Uint8Array; key: Uint8Array },
+  cb: (err: Error | null, buff: Uint8Array | null) => void
+) {
+  if (!global.Buffer) throw Error('Using triplesec currently requires polyfilling `Buffer`');
```

Comment:
`global` is still Node.js specific right? I think [`globalThis`](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/globalThis) is the new-ish environment agnostic global scope, seems to be supported everywhere now.

Reply:
Yes, much better!! 🎉